openvdb_houdini/SOP_NodeVDB.cc

```diff
@@ -708,6 +708,16 @@ OpenVDBOpFactory::~OpenVDBOpFactory()
     if (!mNativeName.empty()) {
         bool invisible = mNativeInvisible;
 
+#ifdef OPENVDB_HOUDINI_HIDE_NATIVE_SOPS
+        // hide the native nodes unless overridden
```

Comment:
As discussed in the TSC meeting, it would be good to log a warning if the environment variable is set but support for it was disabled at compile time.

Reply:
My understanding was that we'd log an error if the opcustomize and the factory compile flag was enabled togther (because they'd both essentially be doing the same thing and that could be confusing).

In my new implementation, I'm using the same environment variable as in your pythonrc file, so there's actually never a situation where support for the environment variable is disabled at compile time.